api.planx.uk/modules/send/idox/nexus.ts

```diff
@@ -227,13 +235,19 @@ async function createSubmission(
   organisationId: string,
   sessionId = "TEST",
 ): Promise<string> {
-  const createSubmissionEndpoint = `${process.env
-    .UNIFORM_SUBMISSION_URL!}/secure/submission`;
+  const createSubmissionEndpoint = `${process.env.IDOX_NEXUS_SUBMISSION_URL!}/secure/submission`;
 
-  const isStaging = ["mock-server", "staging"].some((hostname) =>
+  const isStaging = ["mock-server", "staging", "dev"].some((hostname) =>
     createSubmissionEndpoint.includes(hostname),
   );
 
+  // Get the application type prefix (eg "ldc", "pp", "pa") to send as the "entity"
+  const session = await $api.session.find(sessionId);
+  const rawApplicationType = session?.data.passport.data?.[
```

Comment:
nit: Can we get this more safely and reliably from the DigitalPlanningApplication - I guess it's already zipped at this point and in memory?